pkgs/development/python-modules/slack-sdk/default.nix

```
@@ -54,6 +53,14 @@ buildPythonPackage rec {
     "test_start_raises_an_error_if_rtm_ws_url_is_not_returned"
     # Requires network access: [Errno 111] Connection refused
     "test_send_message_while_disconnection"
+    # [Errno 61] Connection refused
+    "TestWebClient_HttpRetry"
+    # ConnectionResetError: [Errno 54] Connection reset by peer
+    "test_issue_690_oauth_access"
+    "test_issue_690_oauth_v2_access"
+    "test_error_response"
+    # Assertion Error: unexpectedly None (while accessing instance of slack store)
```

```suggestion
    # Requires network access: [Errno 111] Connection refused
    "test_send_message_while_disconnection"
    "TestWebClient_HttpRetry"
    "test_issue_690_oauth_access"
    "test_issue_690_oauth_v2_access"
    "test_error_response"
```
Those are all variants of there is no internet connection.